setup.py

```diff
@@ -2,59 +2,41 @@
 
 import os
 
-# Always prefer setuptools over distutils
 import sys
 
+# Always prefer setuptools over distutils
 from setuptools import find_packages, setup
 
 long_description = open("README.md", "r", encoding="utf-8").read()
 
 setup(
-    name="heareval",
-    version="2021.0.1",
-    description="Holistic Evaluation of Audio Representations (HEAR) 2021 -- Evaluation Kit",
+    name="hearpreprocess",
```

Comment:
The package is still under the dir heareval -- is it in a separate PR that we rename and remove all the heareval code from this?

Reply:
yeah separate PR